nextflow.config

```diff
@@ -8,13 +8,20 @@ process {
     withName: FASTQC {
         cpus = 6
         memory = 8.GB
+        container = 'staphb/fastqc:latest'
```

Comment:
The only suggestion I have here is to generally rely upon the biocontainers project rather.